### ISSUE TITLE
Cannot load "encrypt.dll" module?

### ISSUE THREAD

**Issue by sonichorsee** [tejado/pokemongo-api-demo#197]
After follow all steps installing it, I am running "pokecli.py" on my 64bit windows. When it runs to activate_signature(self, lib_path) and lib_path = "encrypt.dll", I got below error. Any thoughts?

```
self._handle = _dlopen(self._name, mode)
```

WindowsError: [Error 126] The specified module could not be found

It refers to "encrypt.dll" which it say could not be found.

Thanks!

Full message below:

2016-08-06 23:50:41,299 [    pgoapi] [ INFO] pgoapi v1.1.7 - Copyright (c) 2016 tjado https://github.com/tejado
2016-08-06 23:50:42,348 [ utilities] [ INFO] Location for '33.761606, -118.193041' found: 330 S Pine Ave, Long Beach, CA 90802, USA
2016-08-06 23:50:42,348 [ utilities] [ INFO] Coordinates (lat/long/alt) for location: 33.761973 -118.1934017 0.0
2016-08-06 23:50:42,348 [auth_google] [ INFO] Google User Login for: xxxx@gmail.com
2016-08-06 23:50:42,950 [auth_google] [ INFO] Google User Login successful.
2016-08-06 23:50:42,950 [auth_google] [ INFO] Request Google Access Token...
2016-08-06 23:50:43,329 [auth_google] [ INFO] Google Access Token successfully received.
2016-08-06 23:50:43,361 [    pgoapi] [ INFO] Creating a new direct request...
2016-08-06 23:50:43,361 [    pgoapi] [ INFO] Adding 'GET_MAP_OBJECTS' to RPC request including arguments
Traceback (most recent call last):
  File "C:\Users\xxxx\Pokemon\pgoapi-master\pokecli.py", line 133, in <module>
    main()
  File "C:\Users\xxxx\Pokemon\pgoapi-master\pokecli.py", line 128, in main
    response_dict = api.get_map_objects(latitude =position[0], longitude = position[1], since_timestamp_ms = timestamps, cell_id = cell_ids)
  File "C:\Users\xxxx\Pokemon\pgoapi-master\pgoapi\pgoapi.py", line 122, in function
    return request.call()
  File "C:\Users\xxxx\Pokemon\pgoapi-master\pgoapi\pgoapi.py", line 210, in call
    request.activate_signature(lib_path)
  File "C:\Users\xxxx\Pokemon\pgoapi-master\pgoapi\rpc_api.py", line 85, in activate_signature
    self._signature_lib = ctypes.cdll.LoadLibrary(lib_path)
  File "C:\Python27\lib\ctypes__init__.py", line 440, in LoadLibrary
    return self._dlltype(name)
  File "C:\Python27\lib\ctypes__init__.py", line 362, in **init**
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found

**Comment by Vigerus** [comment]
encrypt.dll you downloaded is 32-bit, you probably have 64-bit python environment. Just take the encrypt.c file and create a 64-bit dll, preferably with extern "C" section and encrypt.def:

LIBRARY   encrypt
EXPORTS
   encrypt   @1

Another option is a missing path to dll. Try to place it in your C:\Users\xxxx\Pokemon\pgoapi-master\ and add this path to PATH environment.

**Comment by r0bodog** [comment]
@Vigerus wait, what? I too have a 64-bit python and getting the same error but I don't understand your explanation.

**Comment by elliottcarlson** [comment]
The DLL and SO files are not included in this repo, nor the other repo with the current Unknown6 fixes. To make this work, you need the appropriate file for your systems architecture (32bit vs 64bit, windows vs *nix, etc) and it must be referenced within the code to the appropriate file.

**Comment by sonichorsee** [comment]
Thank you, Elliott. Where can I find all the source code so I can compile
all the Dll I need?

**Comment by elliottcarlson** [comment]
Other repos on Github have it, such as the PokemonGo-Map repo by PokemonGoMap.

**Comment by r0bodog** [comment]
I've managed to get it to work. Thanks!

**Comment by cbogithub** [comment]
http://pgoapi.com/pgoencrypt.tar.gz
